assets/scss/components/_print-link.scss

```diff
@@ -0,0 +1,8 @@
+.print-link {
+  display: inline-block;
+  margin: 0 0 15px -10px;
+  position: relative;
+  padding: 0.5em 0.5em 0.5em 38px;
+  background: url('/assets/images/printer_icon.png') no-repeat 10px 50%;
+  background-size: 16px 18px;
+}
```

Comment:
We have a very similar one here: https://github.com/ministryofjustice/hmpps-prisoner-profile/blob/main/assets/scss/local.scss#L24

Can this be reused?